### PR TITLE
dev-lang/kotlin-bin: new package (v1.4.30)

### PR DESCRIPTION
**Pull request by kma06**

### CODE REVIEW

File: dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+inherit java-pkg-2
+
+DESCRIPTION="Kotlin is a multiplatform, statically typed programming language by JetBrains"
+HOMEPAGE="https://kotlinlang.org"
+SRC_URI="https://github.com/JetBrains/kotlin/releases/download/v${PV}/kotlin-compiler-${PV}.zip"
+
+LICENSE="Apache-2.0 BSD MIT NPL-1.1 Boost-1.0 EPL-1.0 LGPL-2.1"
+SLOT="1.4"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```
Comment (xgqt) [inline dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild:14]: ~amd64 & ~x86, other arches for archtesters

File: dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+EAPI=7
+
+inherit java-pkg-2
+
+DESCRIPTION="Kotlin is a multiplatform, statically typed programming language by JetBrains"
+HOMEPAGE="https://kotlinlang.org"
+SRC_URI="https://github.com/JetBrains/kotlin/releases/download/v${PV}/kotlin-compiler-${PV}.zip"
+
+LICENSE="Apache-2.0 BSD MIT NPL-1.1 Boost-1.0 EPL-1.0 LGPL-2.1"
+SLOT="1.4"
```
Comment (xgqt) [inline dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild:13]: currently unslottable, 1.4 -> 0/1.4

File: dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (xgqt) [inline dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild:4]: eapi 8

File: dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Gentoo Authors
```
Comment (xgqt) [inline dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild:1]: 1999-2022

File: dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+HOMEPAGE="https://kotlinlang.org"
+SRC_URI="https://github.com/JetBrains/kotlin/releases/download/v${PV}/kotlin-compiler-${PV}.zip"
+
+LICENSE="Apache-2.0 BSD MIT NPL-1.1 Boost-1.0 EPL-1.0 LGPL-2.1"
+SLOT="1.4"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+DEPEND="app-arch/unzip"
+RDEPEND=">=virtual/jre-1.8:*"
+BDEPEND=""
```
Comment (xgqt) [inline dev-lang/kotlin-bin/kotlin-bin-1.4.30.ebuild:18]: remove empty BDEPEND

**Comment by xgqt** [comment]
Hi! 
@kma06, are You still interested in being a proxy-maintainer of this? 
I could co-maint this pkg.
In addition to proposed changes, please bump it to 1.7.10 - https://github.com/JetBrains/kotlin/releases/tag/v1.7.10

**Comment by kma06** [comment]
Sure, I haven't had much time to spend with gentoo and I am recently busy with other stuff. I probably won't have time for addressing the reviews.

**Comment by xgqt** [comment]
Ping. You still interested in this pkg?

**Comment by ConiKost** [comment]
Closing. You are welcome to re-open, if there are any news.
@xgqt If you would like to maintain, maybe you could open a new PR?

**Pull request closed without merging**